### ISSUE TITLE
[FEAT] Add support for hosts with path

### ISSUE THREAD

**Issue by milesich** [cars10/elasticvue#19]
**Description**
My elasticsearch server is behind URL with path like `http://example.com/elasticsearch`. When I try to use full url with elasticvue the path is removed: `https://example.com//_search?format=json&size=1000&q=*&from=0&_source=`.

Could you please add support for it?

**Comment by cars10** [comment]
Hi, thanks for the idea!

Can you tell me a bit more about your setup? Do you use a reverse proxy (like `nginx`) to configure the subdirectory?

**Comment by milesich** [comment]
I would like to have both elasticvue and elasticsearch on the same domain so CORS setup in not needed on elasticsearch and also same domain cookies would work. I do a reverse proxy to forward everything with `/elasticsearch` to the elasticsearch server. The rest is server normally as a static content.

**Comment by cars10** [comment]
Thanks, that makes sense. I will try to fix this issue :)

If you can, please share your webserver configuration so i can reproduce and test this setup.

**Comment by milesich** [comment]
I have not tried this since my setup is a bit more complex but this should work:
```
server {
  listen 8080;
  server_name _;
  root /usr/share/nginx/html;

  access_log off;

  location / {
    try_files $uri $uri/ /index.html?$args;
  }

  location /elasticsearch {
    proxy_pass http://127.0.0.1:9200;
    rewrite    ^/elasticsearch(.*) /$1 break;
  }  
}
```

**Comment by cars10** [comment]
Fixed in version `0.21.0`, thanks for your contribution!
